src/Rokt-Kit.js

```diff
@@ -155,6 +155,17 @@ var constructor = function () {
         return replaceOtherWithEmailsha256(userIdentities);
     }
 
+    function returnLocalSessionAttributes() {
+        if (
+            !window.mParticle.Rokt ||
+            typeof window.mParticle.Rokt.getLocalSessionAttributes !==
+                'function'
+        ) {
```

Comment:
We should also add
```
isEmpty(self.placementEventMappingLookup
```

here since there isn't a need to call `window.mParticle.Rokt.getLocalSessionAttributes` if there is no eventMappingLookup